libraries/botbuilder-dialogs/tests/choices_channel.test.js

```diff
@@ -1,103 +1,97 @@
 const assert = require('assert');
 const { Channels } = require('botbuilder-core');
-const { supportsSuggestedActions, supportsCardActions, hasMessageFeed, getChannelId } = require('../lib');
+const { getChannelId, hasMessageFeed, supportsSuggestedActions, supportsCardActions } = require('../');
 
-describe('channel methods', function() {
+describe('channel methods', function () {
     this.timeout(5000);
 
-    it(`should return true for supportsSuggestedActions() with line and 13`, function() {
+    it('should return true for supportsSuggestedActions() with line and 13', function () {
         const validNumOfSuggestedActions = supportsSuggestedActions('line', 13);
-        assert(validNumOfSuggestedActions, `returned false.`);
+        assert(validNumOfSuggestedActions, 'returned false.');
     });
 
-    it(`should return false for supportsSuggestedActions() with line and 14`, function() {
+    it('should return false for supportsSuggestedActions() with line and 14', function () {
         const validNumOfSuggestedActions = supportsSuggestedActions('line', 14);
-        assert(validNumOfSuggestedActions === false, `returned true.`);
+        assert(validNumOfSuggestedActions === false, 'returned true.');
```

Comment:
If I were @joshgummersall, I'd say "since you touched these lines, can you replace them with `assert.strictEqual()`" 😉

Reply:
Heh, fair enough. I'll do a closer pass over the tests and get them up to standard 😄